metabase/metabase
# Migrate timeline event forms to formik

Epic https://github.com/metabase/metabase/issues/26178

Changes:
- Unifies how `useField` is used in wrappers
- Removes `CompactTimeInput` and reverts changes to `TimeInput`
- Adds `FormDateInput` that wraps existing date components
- Adds `EventFrom` and uses it in existing event models
- Removes `FormDateWidget` since it shouldn't be used anywhere else
- Removes old code for forms

How to test:
- Go to a collection, click on the calendar icon in the the header and add an event
- Make sure it is possible to add and edit events via the form

<img width="660" alt="Screenshot 2022-11-07 at 15 29 04" src="https://user-images.githubusercontent.com/8542534/200322722-2f5493a1-211e-4351-9750-d8f423114913.png">
